src/scanner/axe-extension.d.ts

```diff
@@ -31,6 +30,23 @@ declare module 'axe-core/axe' {
         // this must be surrounded by axe.setup and axe.teardown calls
         getSelector: (element: HTMLElement) => string;
     }
+    interface Dom {
+        isVisible: Function;
+        idrefs: (node: HTMLElement, attr: string) => HTMLElement[];
+    }
+    interface Aria {
+        label: Function;
+        implicitRole: Function;
+        getRolesByType: Function;
+        lookupTable: any;
+    }
+
+    interface Text {
+        accessibleText: Function;
+        isHumanInterpretable: Function;
+        sanitize: Function;
+        subtreeText: Function;
+    }
```

Comment:
What impact do these changes have? Was there some kind of error without the interfaces? It seems odd to me that the calls that use these functions didn't need to be updated but I'm not that familiar with setting up a .d.ts file for a library like we've done here.

Reply:
Hi @madalynrose 
Yes correct, we will get errors.
Because for 4.7.2 the above properties were part of axe-extension.d.ts, because in 4.7.2 package these properties were not there. So its added in the code repo.
But in 4.8.4, these properties are added in package, but not fully like few properties are not added fully which is used by our code. So those missing properties are added above.
Please let me know if need further information.

Thanks